html-forms/index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+  <body>
+    <form action="">
+      <h1>Coding Form!</h1>
+      <div>
+        <label for="email">Email</label>
+      </div>
+      <div>
+        <input type="email" name="email" id="email" required>
+      </div>
+      <div>
+        <label for="select">Why is coding cool?</label>
+      </div>
+      <div>
+        <select name="reason">
+          <option value="option">Select an option</option>
+          <option value="interesting">Because it's interesting</option>
+          <option value="fun">Because it's fun</option>
+          <option value="powerful">Because it's powerful</option>
+        </select>
+      </div>
+      <div>
+        <p>Favorite Language (choose one) <br />
+          <input type="radio" name="language" value="html" /> HTML
```

Comment:
There should be a corresponding label for each of these inputs which displays the value